### PR TITLE
Grippers update their icon after attacks with wrapped items

### LINKED ISSUE #9380: Cyborg Gripper Updates Item Sprite Too Early or Not At All
When clicking on an item inside a cyborg gripper, such as a beaker, the sprite will update to how the item should have looked /before/ it was clicked.

For example, with a beaker in the medical gripper:

If the beaker's lid is off, then clicking the sprite will put the lid on the beaker.
However, the beaker's sprite will still show no lid.

If the beaker's lid is on, then clicking the sprite will take the lid off the beaker.
However, the beaker's sprite will now show the lid on the beaker.

I have a feeling that it's updating the gripper's sprite, and _then_ toggling state on the item, rather than the other way around.

---

Another issue with the gripper is, when pouring reagents out of the beaker / bottle, the sprite will not update at all, and the container's sprite will still appear full while it is actually empty.

### PR DESCRIPTION
**Pull request by mikomyazaki**
Fixes #9380

### CODE REVIEW

**Comment by mikomyazaki** [comment]
!bugfix

**Comment by mikomyazaki** [comment]
!review

File: code/_onclick/cyborg.dm [hunk 1]
```diff
@@ -77,6 +77,7 @@
 		//If the gripper contains something, then we will use its contents to attack
 		if (G.wrapped && (G.wrapped.loc == G))
 			GripperClickOn(A, params, G)
+			G.update_icon() //We may need to update our gripper based on a change in the wrapped item
```
Comment (Geevies) [inline code/_onclick/cyborg.dm:80]: wouldn't this be G.wrapped.update_icon() as well? or does the gripper's update_icon handle that

Comment (mikomyazaki) [reply code/_onclick/cyborg.dm:80]: GripperClickOn() handles that, because it calls wrapped.attack() and that already handles all required update_icon()s.

**Pull request merged**